### ISSUE TITLE
Cannot compute MicroProfile properties for the Java element 'Light Parameter'

### ISSUE THREAD

**Issue by kny78** [redhat-developer/intellij-quarkus#414]
I get an issue with IntelliJ and Quarkus-tools plugin:

`Cannot compute MicroProfile properties for the Java element 'Light Parameter'.

java.lang.NullPointerException
	at java.base/java.lang.String.endsWith(String.java:1485)
	at com.redhat.devtools.intellij.quarkus.search.core.utils.AnnotationUtils.isMatchAnnotation(AnnotationUtils.java:61)
	at com.redhat.devtools.intellij.quarkus.search.providers.AbstractAnnotationTypeReferencePropertiesProvider.processAnnotation(AbstractAnnotationTypeReferencePropertiesProvider.java:58)
	at com.redhat.devtools.intellij.quarkus.search.providers.AbstractAnnotationTypeReferencePropertiesProvider.collectProperties(AbstractAnnotationTypeReferencePropertiesProvider.java:49)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.collectProperties(PropertiesManager.java:118)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.lambda$null$1(PropertiesManager.java:96)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.lambda$getMicroProfileProjectInfo$2(PropertiesManager.java:96)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode(DumbService.java:136)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.getMicroProfileProjectInfo(PropertiesManager.java:93)
	at com.redhat.devtools.intellij.quarkus.QuarkusProjectService.lambda$null$4(QuarkusProjectService.java:155)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:959)
	at com.redhat.devtools.intellij.quarkus.QuarkusProjectService.lambda$computeSchema$5(QuarkusProjectService.java:153)
	at com.intellij.openapi.project.DumbServiceImpl.doRun(DumbServiceImpl.java:403)
	at com.intellij.openapi.project.DumbServiceImpl.updateFinished(DumbServiceImpl.java:395)
	at com.intellij.openapi.project.TrackedEdtActivityService$TrackedEdtActivity.run(TrackedEdtActivityService.java:66)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:218)
	at com.intellij.openapi.application.TransactionGuardImpl.access$200(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:200)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:319)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:84)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:133)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:189)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:969)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:839)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)`

**Comment by languitar** [comment]
Happens also with Kotlin:

```
Cannot compute MicroProfile properties for the Java element 'KtUltraLightMethodForSourceDeclaration:products'.

java.lang.NullPointerException
	at java.base/java.lang.String.endsWith(String.java:1485)
	at com.redhat.devtools.intellij.quarkus.search.core.utils.AnnotationUtils.isMatchAnnotation(AnnotationUtils.java:61)
	at com.redhat.devtools.intellij.quarkus.search.providers.AbstractAnnotationTypeReferencePropertiesProvider.processAnnotation(AbstractAnnotationTypeReferencePropertiesProvider.java:58)
	at com.redhat.devtools.intellij.quarkus.search.providers.AbstractAnnotationTypeReferencePropertiesProvider.collectProperties(AbstractAnnotationTypeReferencePropertiesProvider.java:49)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.collectProperties(PropertiesManager.java:118)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.lambda$null$1(PropertiesManager.java:96)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.lambda$getMicroProfileProjectInfo$2(PropertiesManager.java:96)
	at com.intellij.openapi.project.DumbService.lambda$runReadActionInSmartMode$1(DumbService.java:149)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:848)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:61)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode(DumbService.java:142)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.getMicroProfileProjectInfo(PropertiesManager.java:93)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.getMicroProfileProjectInfo(PropertiesManager.java:81)
	at com.redhat.devtools.intellij.quarkus.search.PropertiesManager.getMicroProfileProjectInfo(PropertiesManager.java:72)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$getProjectInfo$4(QuarkusLanguageClient.java:120)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$null$2(QuarkusLanguageClient.java:97)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$3(QuarkusLanguageClient.java:105)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1728)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by kny78** [comment]
Is there any development on this? This bug still annoys me.

**Comment by jeffmaury** [comment]
We're working on it.

**Comment by jeffmaury** [comment]
Fixed by 6b0b7dc73a503a4f110fc5ee0f9146591c719d76
